lib/openstudio-standards/standards/necb/NECB2011/necb_2011.rb

```diff
@@ -1,6 +1,9 @@
 # This class holds methods that apply NECB2011 rules.
 # @ref [References::NECB2011]
 class NECB2011 < Standard
+  require 'zip'
+  require 'open-uri'
```

Comment:
do not add new require statements in the code

Reply:
Which gems require them? I did a `bundle install` and `gem list` and don't see them.

Reply:
So I checked - open-uri was added to the ruby standard library in Ruby 3.0, and zip or rubyzip are not in the standard libraries. I think your options are to switch libraries, or add it to the gemspec. 

It looks like `uri` and `zlib` are default libraries in Ruby 2.7 that you could use instead.

Reply:
Hmmm... after testing, it looks like 'open-uri' and 'zip' are built into the OS 3.6.1 docker container. So I think this is actually good as is.